<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Synkron</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources.qrc" >:/new/prefix1/images/Synkron128.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QWidget" name="mainGridLayout" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>2</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QStackedWidget" name="mainStackedWidget" >
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="page" >
        <widget class="QTabWidget" name="tabWidget" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>637</width>
           <height>361</height>
          </rect>
         </property>
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab" >
          <attribute name="title" >
           <string>Sync #1</string>
          </attribute>
          <attribute name="icon" >
           <iconset resource="resources.qrc" >:/new/prefix1/images/Synkron128.png</iconset>
          </attribute>
          <attribute name="toolTip" >
           <string>Sync #1</string>
          </attribute>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2" >
        <widget class="QWidget" name="restoreGridLayout" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>10</y>
           <width>631</width>
           <height>351</height>
          </rect>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="2" column="0" >
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <widget class="QListWidget" name="restore_list" >
              <property name="statusTip" >
               <string>List of files available for restoring</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <layout class="QGridLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item row="4" column="0" >
               <widget class="QLineEdit" name="path_of_syncfile" >
                <property name="acceptDrops" >
                 <bool>false</bool>
                </property>
                <property name="statusTip" >
                 <string>Original file path</string>
                </property>
                <property name="readOnly" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="0" >
               <widget class="QCheckBox" name="to_black_list" >
                <property name="statusTip" >
                 <string>Add file to blacklist</string>
                </property>
                <property name="text" >
                 <string>Do not synchronise this file again (add to blacklist)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QLabel" name="label_3" >
                <property name="text" >
                 <string>Date and time of synchronisation:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" >
               <widget class="QLabel" name="label_4" >
                <property name="text" >
                 <string>Original file path:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" >
               <widget class="QLabel" name="label_2" >
                <property name="text" >
                 <string>&lt;b>About the selected item:&lt;/b></string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" >
               <widget class="QDateTimeEdit" name="date_of_sync" >
                <property name="statusTip" >
                 <string>Date and time of synchronisation</string>
                </property>
                <property name="readOnly" >
                 <bool>true</bool>
                </property>
                <property name="displayFormat" >
                 <string>yyyy.MM.dd-hh:mm:ss</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" >
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="7" column="0" >
               <layout class="QGridLayout" >
                <property name="margin" >
                 <number>0</number>
                </property>
                <property name="spacing" >
                 <number>6</number>
                </property>
                <item row="0" column="1" >
                 <widget class="QPushButton" name="restore_files" >
                  <property name="statusTip" >
                   <string>Restore selected files</string>
                  </property>
                  <property name="text" >
                   <string>Restore</string>
                  </property>
                  <property name="icon" >
                   <iconset resource="resources.qrc" >:/new/prefix1/images/Restore.png</iconset>
                  </property>
                  <property name="iconSize" >
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" >
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="0" >
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="1" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>&lt;b>Files available for restoring:&lt;/b></string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label_5" >
              <property name="pixmap" >
               <pixmap resource="resources.qrc" >:/new/prefix1/images/Restore16.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="2" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3" >
        <widget class="QWidget" name="blacklistGridLayout" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>20</y>
           <width>651</width>
           <height>341</height>
          </rect>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="6" column="0" >
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="2" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1" >
             <widget class="QPushButton" name="blacklist_removefolder" >
              <property name="statusTip" >
               <string>Remove folder from blacklist</string>
              </property>
              <property name="text" >
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QPushButton" name="blacklist_addfolder" >
              <property name="statusTip" >
               <string>Add folder to blacklist</string>
              </property>
              <property name="text" >
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" >
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="1" >
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>&lt;b>Files in the blacklist:&lt;/b></string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label_8" >
              <property name="pixmap" >
               <pixmap resource="resources.qrc" >:/new/prefix1/images/file.png</pixmap>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0" >
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="2" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1" >
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>&lt;b>Folders in the blacklist:&lt;/b></string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label_9" >
              <property name="pixmap" >
               <pixmap resource="resources.qrc" >:/new/prefix1/images/folder_16.png</pixmap>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0" >
           <widget class="QListWidget" name="blacklist_folderslist" >
            <property name="statusTip" >
             <string>List of folders in the blacklist</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QListWidget" name="blacklist_fileslist" >
            <property name="statusTip" >
             <string>List of files in the blacklist</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="2" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1" >
             <widget class="QPushButton" name="blacklist_removefile" >
              <property name="statusTip" >
               <string>Remove file from blacklist</string>
              </property>
              <property name="text" >
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QPushButton" name="blacklist_addfile" >
              <property name="statusTip" >
               <string>Add file to black list</string>
              </property>
              <property name="text" >
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0" >
           <widget class="Line" name="line" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionNew_sync" />
    <addaction name="actionClose_sync" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionSynchronise" />
    <addaction name="actionRestore" />
    <addaction name="actionBlacklist" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuView" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionNew_sync" />
   <addaction name="actionClose_sync" />
   <addaction name="separator" />
   <addaction name="actionSynchronise" />
   <addaction name="actionRestore" />
   <addaction name="actionBlacklist" />
  </widget>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/new/prefix1/images/STOP.png</iconset>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
   <property name="statusTip" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/new/prefix1/images/Synkron128.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
   <property name="statusTip" >
    <string>View About</string>
   </property>
  </action>
  <action name="actionFolder_1" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/new/prefix1/images/folder.png</iconset>
   </property>
   <property name="text" >
    <string>Set synchronization folder 1</string>
   </property>
  </action>
  <action name="actionFolder_2" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/new/prefix1/images/folder.png</iconset>
   </property>
   <property name="text" >
    <string>Set synchronization folder 2</string>
   </property>
  </action>
  <action name="actionNew_sync" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/new/prefix1/images/Synkron128.png</iconset>
   </property>
   <property name="text" >
    <string>New sync</string>
   </property>
   <property name="statusTip" >
    <string>Open new sync tab</string>
   </property>
  </action>
  <action name="actionClose_sync" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/new/prefix1/images/Synkron128_closetab.png</iconset>
   </property>
   <property name="text" >
    <string>Close sync</string>
   </property>
   <property name="statusTip" >
    <string>Close current sync tab</string>
   </property>
  </action>
  <action name="actionSynchronise" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources.qrc" >:/new/prefix1/images/sync.png</iconset>
   </property>
   <property name="text" >
    <string>Synchronise</string>
   </property>
   <property name="statusTip" >
    <string>Go to Synchronise</string>
   </property>
  </action>
  <action name="actionRestore" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources.qrc" >:/new/prefix1/images/Restore.png</iconset>
   </property>
   <property name="text" >
    <string>Restore</string>
   </property>
   <property name="statusTip" >
    <string>Go to Restore</string>
   </property>
  </action>
  <action name="actionBlacklist" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources.qrc" >:/new/prefix1/images/blacklist.png</iconset>
   </property>
   <property name="text" >
    <string>Blacklist</string>
   </property>
   <property name="statusTip" >
    <string>Go to Blacklist</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>331</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
